src/sentry/services/hybrid_cloud/app/__init__.py

```diff
@@ -1,15 +1,113 @@
 from __future__ import annotations
 
 import abc
+import datetime
 from dataclasses import dataclass, field
-from typing import List
+from typing import TYPE_CHECKING, Any, List, Mapping, Optional, Protocol, TypedDict
 
 from sentry.constants import SentryAppInstallationStatus
+from sentry.models import SentryApp, SentryAppInstallation
 from sentry.services.hybrid_cloud import InterfaceWithLifecycle, silo_mode_delegation, stubbed
+from sentry.services.hybrid_cloud.filter_query import FilterQueryInterface
 from sentry.silo import SiloMode
 
+if TYPE_CHECKING:
+    from sentry.mediators.external_requests.alert_rule_action_requester import AlertRuleActionResult
+
+
+@dataclass
+class RpcSentryAppService:
```

Comment:
I think this is a new concept that doesn't have a directly corresponding model, right? What is it? "Service" is a bit overloaded

Reply:
Unfortunately, this is a `SentryAppService` which is, a, odd class here https://github.com/getsentry/sentry/blob/master/src/sentry/rules/actions/services.py#L33

I'll try to add more comments then explaining.